pkg/apis/pipeline/v1beta1/resource_types.go

```diff
@@ -61,6 +61,7 @@ var AllResourceTypes = resource.AllResourceTypes
 
 // TaskResources allows a Pipeline to declare how its DeclaredPipelineResources
 // should be provided to a Task as its inputs and outputs.
+// Also task-level(Pod) container computing resources can be configured by limits and requests.
```

Comment:
Suggested wording:

`TaskResources is overloaded to represent both PipelineResources and compute resources. PipelineResources are deprecated and will be removed, so this field will only represent compute resources in the future.`